### PR TITLE
DOCSP-42513-correct-source-common-name

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Fixing naming error `migrator.connector.source.common` should be `migrator.connector.sink.common`.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-42513/installation/kafka-deployments/advanced-settings/#sink-connector-properties

## JIRA

https://jira.mongodb.org/browse/DOCSP-42513

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66bbba398383957117cf6fff

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
@tomhollander ready for review.

**Review by tomhollander — CHANGES_REQUESTED** [review]
@alwaysbemark should review this, but my understanding is that both `source` and `sink` settings exist, but we previously had `source` everywhere. The draft change has `sink` everywhere which also isn't correct.

**Comment by ianf-mongodb** [comment]
Ok, I misunderstood the original ask, I have updated only the original section mentioned in the Jira ticket, could you verify this is correct @alwaysbemark? CC: @tomhollander

**Comment by alwaysbemark** [comment]
Tom mentioned the fixes needed, otherwise LGTM!

File: source/installation/kafka-deployments/advanced-settings.txt
Comment (ianf-mongodb) [reply source/installation/kafka-deployments/advanced-settings.txt]: Updated.

File: source/installation/kafka-deployments/advanced-settings.txt
Comment (ianf-mongodb) [reply source/installation/kafka-deployments/advanced-settings.txt]: reverted

**Review by tomhollander — APPROVED** [review]

**Pull request merged**
